qiskit_ibm_runtime/utils/backend_decoder.py

```diff
@@ -66,6 +74,43 @@ def configuration_from_server_data(
     return None
 
 
+def filter_raw_configuration(
+    raw_config: dict, use_fractional_gates: Optional[bool] = False
```

Comment:
I am not sure which interface you are saying you prefer to keep. `QiskitRuntimeService.backend` (and `QiskitRuntimeService.backends`) only accepted `use_fractional_gates` before and they still have three behaviors depending on whether `use_fractional_gates` is set to `True`, `False`, or `None`. I had been thinking that was the only public interface related `use_fractional_gates` (besides the `use_fractional_gates` backend option, which I think can still be modified as long as you use `IBMBackend.refresh()` afterward).

Reply:
I think you are referring to the two flags of `convert_to_target`. I don't think we care about removing both fractional gates and dynamic circuit instructions from the backend, so that leaves only the three options covered by the `True`, `False`, and `None` values of `use_fractional_gates`. Those options are a little weird versus explicit `include_fractional_gates` and `include_control_flow` options but that is the way it already works for `QiskitRuntimeService.backend()`.

If your concern here is just related to the change to `convert_to_target()`, perhaps we should move the discussion to your other comment there.

Reply:
Yes, the two flags are what I'm referring to. So you suggest an implicit mapping of

- `include_control_flow=True, include_fractional_gates =True` := `use_fractional_gates =None`
- `include_control_flow=True, include_fractional_gates =False` := `use_fractional_gates =True`
- `include_control_flow=False, include_fractional_gates =True` := `use_fractional_gates =False`

Maybe this is okey but the behavior with `use_fractional_gates =None` is not straightforward, not documented and not tested. So there is no guarantee this behavior is preserved for future release. At least test and document should be added.

Another concern is potential future upgrade of our instruction set. If we need to deal with third kind of instruction group, single flag is not enough and we would need breaking API change. Maybe this is over anxious though.

Reply:
Maybe new implementation is okey with update of documentation.

Reply:
Yes, the `use_fractional_gates=None` behavior was already present before this PR. It was added in https://github.com/Qiskit/qiskit-ibm-runtime/pull/1938. It is documented in the `IBMBackend.backend()` documentation and it is tested, so I think we are okay here. In any case, I restored the options to `convert_to_target`.

Reply:
Yes, I think we are good -- I put back the two options to `convert_to_target` and at least didn't make the situation with the three modes of `use_fractional_gates` worse (though I agree it is not the most clear way of working).